pep-0725.rst

```diff
@@ -0,0 +1,599 @@
+also present in a number of packaging systems - but not always, and the details
+of their implementation varies. 
+
+Cross compilation
+-----------------
+
+Cross compilation is not yet (as of August 2023) well-supported by stdlib
+modules and ``pyproject.toml`` metadata. It is however important when
+translating external dependencies to those of other packaging systems (with
+tools like ``pyp2rpm``). Introducing support for cross compilation immediately
+in this PEP is much easier than extending ``[external]`` in the future, hence
+the authors choose to include this now.
+
+Terminology
+'''''''''''
+
+This PEP uses the following terminology:
+
+- *build machine*: the machine on which the package build process is being
+  executed
+- *host machine*: the machine on which the produced artifact will be installed
+  and run
+- *build dependency*: dependency for building the package that needs to be
+  present at build time and itself was built for the build machine's OS and
+  architecture
+- *host dependency*: dependency for building the package that needs to be
+  present at build time and itself was built for the host machine's OS and
+  architecture
+
+Note that this terminology is not consistent across build and packaging tools,
+so care must be taken when comparing build/host dependencies in
+``pyproject.toml`` to dependencies from other package managers.
+
+Note that "target machine" or "target dependency" is not used in this PEP. That
+is typically only relevant for cross-compiling compilers or other such advanced
+scenarios [#gcc-cross-terminology]_, [#meson-cross]_ - this is out of scope for
+this PEP.
+
+Finally, note that while "dependency" is the term most widely used for packages
+needed at build time, the existing key in ``pyproject.toml`` for PyPI
+build-time dependencies is ``build-requires``. Hence this PEP uses the keys
+``build-requires`` and ``host-requires`` under ``[external]`` for consistency.
+
+Build and host dependencies
+'''''''''''''''''''''''''''
+
+Clear separation of metadata associated with the definition of build and target
+platforms, rather than assuming that build and target platform will always be
+the same, is important [#pypackaging-native-cross]_.
+
+Build dependencies are typically run during the build process - they may be
+compilers, code generators, or other such tools. In case the use of a build
+dependency implies a runtime dependency, that runtime dependency does not have
+to be declared explicitly. For example, when compiling Fortran code with
+``gfortran`` into a Python extension module, the package likely incurs a
+dependency on the ``libgfortran`` runtime library. *Rationale for not
+explicitly listing such runtime dependencies: (a) it may depend on
+compiler/linker flags or details of the build environment whether the
+dependency is present, and (b) these runtime dependencies can be detected and
+handled automatically by tools like* ``auditwheel``.
```

Comment:
Is this entire stanza intended to be in italics? (Perhaps it could be a footnote?)

Reply:
You're right, best left out - I removed the italics and reworded it slightly.

Thanks for the review @AA-Turner